src/Structures/MDClient.js

```diff
@@ -29,6 +28,10 @@ module.exports = class MenuDocsClient extends Client {
 		if (!options.prefix) throw new Error('You must pass a prefix for the client.');
 		if (typeof options.prefix !== 'string') throw new TypeError('Prefix should be a type of String.');
 		this.prefix = options.prefix;
+		
+		if (!options.owners) throw new Error('You must pass a list of owners for the client.');
+		if (typeof options.prefix !== 'string') throw new TypeError('Owners should be a type of Array<String>.');

```

Comment:
options.prefix?

Reply:
Prefix doesn't have to be a type of `Array<string>` always, you can fix this by checking if prefix is a type of Array then have it or else return the prefix in a type of Array, this allows to have both string and Array<string>

```js
// This will always be a array.
this.prefix = Array.isArray(options.prefix) ? options.prefix : [options.prefix];
```

Reply:
Although that's a great addition, I was actually pointing out the fact that he's now double checking the prefix type when in reality on that line he should be checking the owners type.